src/System.Private.CoreLib/shared/System/Threading/Timer.cs

```diff
@@ -64,53 +66,7 @@ private static TimerQueue[] CreateTimerQueues()
 
         #region interface to native per-AppDomain timer
```

Comment:
It may be nice to remove references to AppDomain.

Reply:
And change the `m_` prefix to `_` to make the style uniform and match coding conventions.